### PR TITLE
Backend boilerplate

### PR DESCRIPTION
**Pull request by jkhirano**
Created server, models, decorators.
Model and decorators not testable yet.

- [x] cd into root
- [x] test server by nodemon server/server.js
- [x] postman - get localhost:8080
- [x] test should say "hiyee"

### CODE REVIEW

File: server/database/models/Item.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    return true;
+  }
+  user() {
+    return this.hasOne("User");
+  }
+  images() {
+    return this.belongsToMany("Image");
+  }
+  category() {
+    return this.hasOne("category");
```
Comment (rianmueller) [inline server/database/models/Item.js]: Instead of "category", should this be "ItemCategory"?

File: server/database/models/Item.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+  condition() {
+    return this.hasOne("condition");
```
Comment (rianmueller) [inline server/database/models/Item.js]: Should this be "ItemCondition"?

**Review by andrewobx — APPROVED** [review]
Lookin good

File: server/database/models/Item.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  status() {
+    return this.hasOne("status");
```
Comment (rianmueller) [inline server/database/models/Item.js]: Should this be "ItemStatus"?

File: server/database/models/User.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  }
+  get hasTimestamps() {
+    return true;
+  }
+  items() {
+    return this.belongsToMany("Item");
+  }
+
+  userStatus() {
+    return this.hasOne("userStatus");
```
Comment (rianmueller) [inline server/database/models/User.js]: Should this be "UserStatus"?

**Review by rianmueller — CHANGES_REQUESTED** [review]

**Review by rianmueller — APPROVED** [review]
lgtm

**Pull request merged**